addon/lib/Pioneer.jsm

```diff
@@ -0,0 +1,30 @@
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(EXPORTED_SYMBOLS|Pioneer)" }]*/
+
+const { utils: Cu } = Components;
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+
+XPCOMUtils.defineLazyModuleGetter(
+  this, "Config", "resource://esper-pioneer-shield-study/Config.jsm"
+);
+XPCOMUtils.defineLazyModuleGetter(
+  this, "PioneerUtils", "resource://esper-pioneer-shield-study/PioneerUtils.jsm"
+);
+
+const Pioneer = {
+  async startup(addonData) {
+    this.utils = new PioneerUtils(Config);
+    const branch = await Pioneer.utils.chooseBranch();
```

Comment:
do you need this if there is only one branch?

Reply:
Not in this addon, but always trying to write in a re-usable way. Also, this code ought to be in pioneer utils and always sent as metadata regardless of if the study/addon only uses one branch.